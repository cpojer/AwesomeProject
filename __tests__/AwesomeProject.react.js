global.__DEV__ = false;
const React = require('React');
let AwesomeProject;
const ReactNativeMock = require('../test/ReactNativeMock');
jest.unmock('react/lib/ReactTestRenderer');
const renderer = require('react/lib/ReactTestRenderer');
describe('AwesomeProject', () => {
    
    beforeEach (() => {        
        jest.setMock('react-native', ReactNativeMock);
        AwesomeProject = require('../AwesomeProject.react');
    })
    
    it('should work', () => {
      const example = <AwesomeProject />;      
      const tree = renderer.create(example);
      expect(tree.toJSON()).toMatchSnapshot();  
    })

})