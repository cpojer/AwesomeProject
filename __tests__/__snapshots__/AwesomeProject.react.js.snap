exports[`AwesomeProject should work 0`] = `<view
  style={
    Object {
      "flex": 1,
      "justifyContent": "center",
      "alignItems": "center",
      "backgroundColor": "#F5FCFF"
    }
  }>
  <text
    style={
      Object {
        "fontSize": 20,
        "textAlign": "center",
        "margin": 10
      }
    }>
    Welcome to React Native!
  </text>
  <text
    style={
      Object {
        "textAlign": "center",
        "color": "#333333",
        "marginBottom": 5
      }
    }>
    To get started, edit index.ios.js
  </text>
  <text
    style={
      Object {
        "textAlign": "center",
        "color": "#333333",
        "marginBottom": 5
      }
    }>
    Press Cmd+R to reload,
    
    
    Cmd+D or shake for dev menu
  </text>
</view>`;
